work-scheduler-web/pom.xml

```diff
@@ -25,6 +25,7 @@
         <version.jstl>1.2</version.jstl>
         <version.plugin.war>3.1.0</version.plugin.war>
         <version.plugin.jetty>9.4.6.v20170531</version.plugin.jetty>
+        <version.hibernate-validator>5.3.5.Final</version.hibernate-validator>
```

Comment:
You already have the validator version property in your parent POM, you don't need this here.